### DISCUSSION TITLE
should provide an option to connect without SIWE

### DISCUSSION THREAD

**Discussion by shrugs** [family/connectkit#147] [Q&A] [answered]
in my app, i've integrated SIWE and CK but there exists situations where i would like the user to connect, but not necessarily authenticate. in our app we allow bidding on an nft, which only requires connecting a wallet and sending a tx, and chatting, which obviously requires authentication. i'd like to provide an in-context "connect wallet" button within the bidding ui that does _not_ ask the user to authenticate with SIWE, to reduce steps to tx.

**Comment by ikeadrift** [comment, marked as answer]
solved with https://github.com/family/connectkit/commit/79814fbfd51b1c1f58c2b9db32a9465934969876

**Reply by lochie** [reply]
🫡
